### DISCUSSION TITLE
backgroundImage extend does not effect

### DISCUSSION THREAD

**Discussion by HurricaneTBS** [tailwindlabs/tailwindcss#3540] [Help]
![image](https://user-images.githubusercontent.com/46099292/107147742-a5067800-698a-11eb-80f8-9fc49a3309a5.png)
![image](https://user-images.githubusercontent.com/46099292/107147748-acc61c80-698a-11eb-9c25-3df0314896cc.png)
it does not work

**Comment by simonswiss** [comment]
Hey!

This works properly, so the issue you have probably has to do with the `path` you have for the image in your Tailwind config file:

https://play.tailwindcss.com/cfVp1Eko5X

![image](https://user-images.githubusercontent.com/485747/107165080-d7a18680-6a05-11eb-95c4-b11385b3c27a.png)

Did you place that image in your `public` directory in the Vue project?
